src/form.js

```diff
@@ -353,6 +353,16 @@ define([
                 return m.p.nodeID === nodeID;
             });
         },
+        getMugChildrenByChoiceValue: function (mug, nodeID) {
+            var parentNode = (mug ? this.dataTree.getNodeFromMug(mug)
+                                  : this.dataTree.rootNode);
+            return _.filter(parentNode.value._node_control.children, function (m) {
```

Comment:
Did you consider `_.filter(this.getChildren(mug), ...` instead of the above three lines?